processYear.ts

```diff
@@ -315,6 +315,14 @@ const selectClusters = async (
         // results.skippedClusters.push(cluster); // keeping for testing for now
         break;
       } else {
+        const distanceToFacility = getDistance(
+          { lat: params.facilityLat, lng: params.facilityLng },
```

Comment:
do we want to do `params.lat` instead of `params.facilityLat`?  When we do the radius it's not necessarily from the facility itself, if they select a different biomass gather point.